example/e2e/utils/matchers.js

```diff
@@ -6,7 +6,7 @@ const elementByText = (text) => element(by.text(text));
 const getDateTimePickerIOS = () =>
   element(by.type('UIPickerView').withAncestor(by.id('dateTimePicker')));
 
-const getInlineTimePickerIOS = () => element(by.label('Time Picker'));
+const getInlineTimePickerIOS = () => element(by.type('RNDateTimePicker'));
```

Comment:
just wondering, why this change?

Reply:
I think we can revert this. This was needed before 93278605fcd14bdb735b015e9f725f2c21469e23. The additional `<View>` Tag got the label `Time Picker`. Detox found two elements with the same label and failed.

Reply:
```suggestion
const getInlineTimePickerIOS = () => element(by.label('Time Picker'));
```